### ISSUE TITLE
Can't add Let's Encrypt to Domain

### ISSUE THREAD

**Issue by xcrap** [plesk/letsencrypt-plesk#172]
I had never problems since several months with the plugin, I have almost 30 domains with the certificate but after two weeks I can't install certificates anymore, I got this:

Error: Let's Encrypt SSL certificate installation failed: Challenge marked as invalid. Details: Could not connect to hmt.pt

Checking the logs: AH01909: RSA certificate configured for hmt.pt:443 does NOT include an ID which matches the server name

I  have made no changes on the server unless the regular Plesk Updates....

**Comment by xgin** [comment]
Seems yet another IPv6 issue... see #171 
https://www.ssllabs.com/ssltest/analyze.html?d=hmt.pt&latest - also reports "Unable to connect to the server"

**Comment by xcrap** [comment]
Yes I've removed ipv6, connected Let's Encrypt and added ipv6 again, not sure what's wrong with those settings as it worked before. Will check with the hosting. Thanks.

**Comment by xgin** [comment]
Take into account that DNS has TTL and your changes are not applied immediately.
Let us know if the problem is solved or not.

**Comment by xcrap** [comment]
I've disabled the IPV6, activated Let's Encrypt successfully and enabled again IPV6. The hosting guys are investigating if something wrong with out IPV6 but they say no but it appears otherwise, Still waiting for their answer.
